packages/contact-center/task/src/Utils/task-util.ts

```diff
@@ -495,7 +498,7 @@ export function getControlsVisibility(
         isConsultInProgress,
         isCustomerInCall,
         conferenceParticipantsCount,
-        maxParticipantsInConference,
+        MAX_PARTICIPANTS_IN_MULTIPARTY_CONFERENCE,
```

Comment:
I believe all this will be in the future abstracted into the SDK with Task Refactor item. Kindly confirm